src/engine/renderer/tr_init.cpp

```diff
@@ -179,6 +179,11 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 	cvar_t      *r_lockpvs;
 	cvar_t      *r_noportals;
 	cvar_t      *r_portalOnly;
+
+	/* Quake3 used 256 as default portal range. */
+	Cvar::Modified<Cvar::Cvar<int>> r_portalDefaultRange(
+		"r_portalDefaultRange", "Default portal range", CVAR_LATCH | CVAR_ARCHIVE, 1024 );
```

Comment:
You have to use `Cvar::LATCH` with new style

Otherwise LGTM

Reply:
This way?

```c++
	Cvar::Modified<Cvar::Cvar<int>> r_portalDefaultRange(
		"r_portalDefaultRange", "Default portal range", Cvar::LATCH | Cvar::ARCHIVE, 1024 );
```

Reply:
Yeah. Although I'm not sure why it needs to be archive :man_shrugging:

Reply:
I removed the `ARCHIVE` flag.